scripts/nginx/nginx.conf

```diff
@@ -0,0 +1,122 @@
+user  www-data;
+
+worker_processes  1;
+
+error_log  logs/error.log;
+#error_log  logs/error.log  notice;
+#error_log  logs/error.log  info;
+
+#pid        logs/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+http {
```

Comment:
This may be best to implement in a follow up PR, but any reason we need to support http here? Can we just redirect this to https?

Reply:
http handles the 301 redirect to https